### PR TITLE
Fix Google Branding logo according to Guidelines.

### PR DESCRIPTION
**Pull request by manjotsidhu**
Fixes Google Social login according to the Google Guidelines published : [Here](https://developers.google.com/identity/branding-guidelines#top_of_page)

#### Describe the changes you have made in this PR -
Added google branded SVG that follows Google Guidelines.

### Screenshots of the changes (If any) -
image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/views/users/shared/_links.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
     </div>
     <div class="row center-row">
       <a href="/users/auth/google_oauth2">
-        <%= image_tag "SVGs/google-plus.svg", alt: "Google Icon", :class => "users-social-links" %>
+        <%= image_tag "SVGs/google_login.svg", alt: "Google Icon", :class => "users-social-links", style: 'width:auto;'%>
```
[the remark this answers is not in the document]
Comment (manjotsidhu) [reply app/views/users/shared/_links.html.erb:8]: Sure.

**Comment by satu0king** [comment]
Please follow the guidelines here: https://developers.google.com/identity/branding-guidelines#top_of_page

**Comment by manjotsidhu** [comment]
> Please follow the guidelines here: https://developers.google.com/identity/branding-guidelines#top_of_page

I think I followed the guidelines, the branded logo of Google is required and the keyword "Google" must be there.

**Comment by manjotsidhu** [comment]
> > Please follow the guidelines here: https://developers.google.com/identity/branding-guidelines#top_of_page
> 
> I think I followed the guidelines, the branded logo of Google is required and the keyword "Google" must be there.

image
This is the one I used.

**Comment by satu0king** [comment]
@aayush-05 please review this PR, merge this or #1918

**Comment by manjotsidhu** [comment]
> @aayush-05 please review this PR, merge this or #1918

Looks like he is busy.

**Comment by aayush-05** [comment]
@satu0king sorry for the delay, reviewed #1918 (that one looks better). Once the changes are made, we can merge that. This one can be closed

**Pull request closed without merging**